tests/analytics_loaded_fields.txt

```diff
@@ -0,0 +1,76 @@
+*_ancestor_path
```

Comment:
This file is surely generated everytime, so it shouldn't need to be in version control.